.vscode/settings.json

```diff
@@ -0,0 +1,5 @@
+{
+  "editor.defaultFormatter": "esbenp.prettier-vscode",
+  "editor.formatOnSave": true,
+  "editor.tabSize": 2
```

Comment:
do we need this here? it is already defined in `.prettierrc` file.

Reply:
tabsize removed and the other settings it is instead of making the vscode settings for this project only and not globally